### PR TITLE
Changes for OpenShift

### PR DESCRIPTION
**Pull request by jpkrohling**

### CODE REVIEW

**Comment by jpkrohling** [comment]
@objectiser , @pavolloffay , could you please review/merge?

**Comment by objectiser** [comment]
@jpkrohling Tested locally and it seems fine. :+1:

File: tests/app/vertx-opentracing/Dockerfile [hunk 1]
```diff
@@ -21,8 +21,10 @@ MAINTAINER Gary Brown <gbrown@redhat.com>
 
 ENV APP_HOME /app/
 
-COPY hawkular-apm-tests-app-vertx-opentracing-fat.jar $APP_HOME
+COPY target/hawkular-apm-tests-app-vertx-opentracing-fat.jar $APP_HOME
 
 WORKDIR $APP_HOME
+RUN chmod 777 /app/
```
Comment (pavolloffay) [inline tests/app/vertx-opentracing/Dockerfile]: Could you please use  `$APP_HOME` variable?

**Comment by objectiser** [comment]
retest this please

**Comment by objectiser** [comment]
@jpkrohling Thanks!

**Pull request merged**